src/index.html

```diff
@@ -12,6 +12,8 @@
 
 	<!-- inject:css -->
 	<!-- endinject -->
+
+	<script src="/js/app.config.js"></script>
```

Comment:
That's right! I knew that was a smelly LOC! :hankey: 

I'll think about it again tomorrow.